python/pyarrow/tests/test_json.py

```diff
@@ -357,3 +608,17 @@ def read_json(self, *args, **kwargs):
         table = read_json(*args, **kwargs)
         table.validate(full=True)
         return table
+
+
+class TestSerialStreamingJSONRead(BaseTestStreamingJSONRead, unittest.TestCase):
+
+    @property
+    def use_threads(self):
+        return False
```

Comment:
Nit: I think this can simply be:
```suggestion
    use_threads = False
```